sites/partners/src/components/listings/PaperListingDetails/sections/DetailUnits.tsx

```diff
@@ -60,6 +60,13 @@ const DetailUnits = ({ setUnitDrawer }: DetailUnitsProps) => {
 
   return (
     <SectionWithGrid heading={t("listings.units")} inset>
+      {listing.homeType && (
```

Comment:
I don't want to show the home type if jurisdiction doesn't have it enabled. I have updated this so that it checks if the feature flag is turned on, if so it then will show "none" if no home type is selected.